app/src/components/textForm.js

```diff
@@ -1,8 +1,8 @@
-export default function TextForm({type, placeholder, onChange, value=""}) {
+export default function TextForm({type, placeholder, onChange, pattern, min, max, step, value=null}) {
```

Comment:
What's is pattern and step for? I don't see these parameters being used when creating a new textform